public/data/deepanshu1334.json

```diff
@@ -0,0 +1,33 @@
+{
+    "name": "Deepanshu kumar",
+    "type": "personal",
+    "bio": "Open Source Enthusiast",
+    "avatar": "https://github.com/Deepanshu1334.png",
+    "links": [
+      {
+        "name": "Follow me on GitHub",
+        "url": "https://github.com/Deepanshu1334",
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/Deepans23454090",
+        "icon": "twitter"
+      },
+      {
+        "name": "Follow me on Hashnode",
+        "url": "https://deepanshu669.hashnode.dev",
+        "icon": "hashnode"
+      },
+      {
+        "name": "Follow me on medium",
+        "url": "https://medium.com/@deepanshuchaudhry852",
+        "icon": "medium"
+      },
+      {
+        "name": "Follow me on linkedIn",
```

Comment:
I recommend capitalizing the `l` here. What do you think?
```suggestion
        "name": "Follow me on LinkedIn",
```


__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question&nbsp; | &nbsp;**Tags:** Inelegant